docs/changes/utils/13311.feature.rst

```diff
@@ -0,0 +1,3 @@
+A new decorator -- ``on_metaclass`` -- has been added to make a method
+behave as if it were defined on the metaclass, not the class. This can be
+combined with other decorators like ``classmethod`` and ``property``.
```

Comment:
Saying that the new decorator allows for creating class-only methods that can be accessed from a class, but not its instances, might be more understandable for more people.

Reply:
SGTM. Also changing the second sentence since https://stackoverflow.com/questions/128573/using-property-on-classmethods/64738850#64738850